account_analytic_post_by_type/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
```

Comment:
Can you add a contributors section to the readme and put SerpentCS there and not here?